spec/presenters/two_factor_authentication/selection_presenter_spec.rb

```diff
@@ -7,7 +7,9 @@ def method
     end
   end
 
-  subject(:presenter) { described_class.new }
+  attr_reader :configuration, :user
```

Comment:
I would not expect to see `attr_reader` in specs.

Reply:
refactored that at cf44f91